packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -35,12 +37,15 @@ const getMarketsParamsSpecific = t.intersection([
     creator: t.string,
     category: t.string,
     search: t.string,
-    reportingState: t.union([t.string, t.array(t.string)]),
+    reportingStates: t.array(t.string),
     disputeWindow: t.string,
```

Comment:
is dispute window the address of a dispute window?

Reply:
This was left in because we were trying to keep the getters as similar to the getters from v1. If disputeWindow isn't very useful as a param, I'll remove it.

Reply:
Yes remove it.